ios/Video/RCTVideo.swift

```diff
@@ -312,7 +312,7 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
 
     // MARK: - Progress
 
-    func sendProgressUpdate() {
+    func sendProgressUpdate(fromEnd: Bool = false) {
```

Comment:
For me `didEnd` is better name for this, WDYT ?
```suggestion
    func sendProgressUpdate(didEnd: Bool = false) {
```

Reply:
Yes you are right ! done